### ISSUE TITLE
Out-of-memory

### ISSUE THREAD

**Issue by talex5** [talex5/qubes-mirage-firewall#5]
```
2016-03-09 23:21.27: INF [memory_pressure] Writing meminfo: free 4068 / 18012 kB (22.58 %)
--- End dump ---
Fatal error: exception Out of memory
Raised by primitive operation at file "lib/gnt.ml", line 76, characters 20-74
Called from file "lib/backend.ml", line 210, characters 26-60
Called from file "lib/backend.ml", line 224, characters 6-44
Called from file "src/core/lwt.ml", line 882, characters 20-24
Mirage exiting with status 2
Do_exit called!
```

**Comment by talex5** [comment]
I haven't seen any OOM errors for a few months now, but keeping this open because we really should use a fixed-size NAT table (rather than catching `Out_of_memory` and dropping the table from time-to-time). See: https://github.com/yomimono/mirage-nat/issues/17

**Comment by talex5** [comment]
Was fixed in v0.3.
